src/webgpu/api/validation/createPipelineLayout.spec.ts

```diff
@@ -339,8 +367,9 @@ g.test('bind_group_layouts,create_pipeline_with_null_bind_group_layouts')
           code,
         });
 
+        const format = 'rgba8unorm';
         t.expectValidationError(() => {
```

Comment:
How about rename line 349 `shouldError` -> `shouldCreatePipelineError`, and make the expectation like
``` typescript
        t.expectValidationError(() => {
          const renderPipeline = t.device.createRenderPipeline({
            layout,
            vertex: {
              module: shaderModule,
            },
            fragment: {
              module: shaderModule,
              targets: [
                {
                  format,
                },
              ],
            },
          });
        }, shouldCreatePipelineError);

        // If pipeline is created successfully, pass encoder can use it to test setBindGroup, 
        // which should always success.
        if (!shouldCreatePipelineError) {
          const renderTarget = t.createTextureTracked({
            usage: GPUTextureUsage.RENDER_ATTACHMENT | GPUTextureUsage.COPY_SRC,
            size: [4, 4, 1],
            format,
          });
          const commandEncoder = t.device.createCommandEncoder();
          const renderPassEncoder = commandEncoder.beginRenderPass({
            colorAttachments: [
              {
                view: renderTarget.createView(),
                loadOp: 'load',
                storeOp: 'store',
              },
            ],
          });
          for (let i = 0; i < 4; ++i) {
            if (!setBindGroupOnEmptyBindGroupLayoutIndex && i === emptyBindGroupLayoutIndex) {
              continue;
            }
            renderPassEncoder.setBindGroup(i, bindGroups[i]);
          }
          renderPassEncoder.setPipeline(renderPipeline);
          renderPassEncoder.draw(1);
          renderPassEncoder.end();

          t.queue.submit([commandEncoder.finish()]);
        }

        break;
```